pkg/providers/aws/provider_redis.go

```diff
@@ -620,17 +620,19 @@ func buildRedisGenericMetricLabels(r *v1alpha1.Redis, clusterID, cacheName strin
 	labels["instanceID"] = cacheName
 	labels["productName"] = r.Labels["productName"]
 	labels["strategy"] = redisProviderName
+	labels["statusAWS"] = ""
+	labels["statusPhase"] = ""
 	return labels
 }
 
 // adds extra information to labels around resource
 func buildRedisInfoMetricLables(r *v1alpha1.Redis, group *elasticache.ReplicationGroup, clusterID, cacheName string) map[string]string {
 	labels := buildRedisGenericMetricLabels(r, clusterID, cacheName)
-	if group != nil {
-		labels["status"] = *group.Status
+	if group != nil && len(string(r.Status.Phase)) != 0 {
+		labels["statusAWS"] = *group.Status
+		labels["statusPhase"] = string(r.Status.Phase)
 		return labels
 	}
-	labels["status"] = fmt.Sprintf("%s is nil", cacheName)
```

Comment:
Same questions/comments here as with the postgres provider

Reply:
restoring this function and moving phase status to new function